Sources/SourceControl/GitRepository.swift

```diff
@@ -82,28 +82,46 @@ public struct GitRepositoryProvider: RepositoryProvider, Cancellable {
         self.git = GitShellHelper(cancellator: cancellator)
     }
 
+    private func clone(_ repository: RepositorySpecifier,
```

Comment:
can we move this function after the "fetch" function, so the callGit are the first functions

Reply:
Sure